nano/node/confirming_set.hpp

```diff
@@ -20,6 +21,14 @@
 
 namespace mi = boost::multi_index;
 
+namespace nano
+{
+class election;
+class ledger;
+class logger;
+class stats;
```

Comment:
This could be in fwd header to avoid constantly repeating the same class, it's just noise

Reply:
Good point. Moved all of these forward declarations plus some other opportunistic ones in to the fwd.hpp files you started.